### ISSUE TITLE
Automate Harvard SCW collection

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme-airflow#536]
Automate in airflow, if possible, or work up a manual solution that converts to json outside of traject.

**Comment by aaron-collier** [comment]
@jacobthill is this collection different from the harvard ihp collection which uses the oai_xml driver? If so, do you have an example config or documentation from harvard on the format of the above URL?

**Comment by jacobthill** [comment]
Yes it is a different server and metadata format. I haven't found any documentation but it serves mods data and the API returns some info. e.g. you can set the limit like so https://api.lib.harvard.edu/v2/items?q=%E2%80%9CStuart%20Cary%20Welch%20Islamic%20and%20South%20Asian%20Photograph%20Collection.%E2%80%9D&limit=250.

I think this is just a pagination issue.
